src/main/java/com/monumental/models/Image.java

```diff
@@ -0,0 +1,44 @@
+package com.monumental.models;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+/**
+ * Model class for an Image
+ * Contains all of the state related to an Image and Getters and Setters for that state
+ * Contains a many-to-one relationship with Monument
+ */
+
+@Entity
+@Table(name = "image", uniqueConstraints = {
+        @UniqueConstraint(columnNames = "id")
+})
+public class Image extends Model implements Serializable {
+
+    @Column(name = "url")
+    private String url;
+
+    @ManyToOne
```

Comment:
We should use lazy loading on these since we won't always need them
```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```